solidity/contracts/converter/BancorConverter.sol

```diff
@@ -47,15 +41,16 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
     */
     uint16 public version = 27;
 
-    IWhitelist public conversionWhitelist;          // whitelist contract with list of addresses that are allowed to use the converter
-    IERC20Token[] public reserveTokens;             // ERC20 standard token addresses (prior version 17, use 'connectorTokens' instead)
-    mapping (address => Reserve) public reserves;   // reserve token addresses -> reserve data (prior version 17, use 'connectors' instead)
-    uint32 public totalReserveRatio = 0;            // total ratio of all reservces, also used to efficiently prevent increasing
-                                                    // the total reserve ratio above 100%
-    uint32 public maxConversionFee = 0;             // maximum conversion fee for the lifetime of the contract,
-                                                    // represented in ppm, 0...1000000 (0 = no fee, 100 = 0.01%, 1000000 = 100%)
-    uint32 public conversionFee = 0;                // current conversion fee, represented in ppm, 0...maxConversionFee
-    bool public conversionsEnabled = true;          // deprecated, backward compatibility
+    IWhitelist public conversionWhitelist;                  // whitelist contract with list of addresses that are allowed to use the converter
+    IERC20Token[] public reserveTokens;                     // ERC20 standard token addresses (prior version 17, use 'connectorTokens' instead)
+    mapping (address => Reserve) public reserves;           // reserve token addresses -> reserve data (prior version 17, use 'connectors' instead)
+    mapping (address => uint256) private reserveBalances;   // reserve token addresses -> reserve balance

```

Comment:
Since it's not supposed to be public information. The getter function is mostly there in order to be able to override it as well as support a cleaner implementation once we have virtual balance back in the contract.
But users aren't supposed to query the storage balance by accessing that var.